code/modules/religion/burdened/psyker.dm

```diff
@@ -181,14 +182,34 @@
 		"Daredevil" = "Hey now, you won't be reckless with this, will you?",
 		"Lacytanga" = "Rules are written by the strong.",
 		"A10" = "The fist of God. Keep away from the terrible.",
+		"Lucky" = "Ain't that a kick in the head?",
 	)
 
 /obj/item/gun/ballistic/revolver/chaplain/Initialize(mapload)
 	. = ..()
-	AddComponent(/datum/component/anti_magic, MAGIC_RESISTANCE_HOLY)
+	AddComponent(/datum/component/anti_magic, MAGIC_RESISTANCE|MAGIC_RESISTANCE_HOLY)
+	AddComponent(/datum/component/effect_remover, \
+		success_feedback = "You disrupt the magic of %THEEFFECT with %THEWEAPON.", \
+		success_forcesay = "BEGONE FOUL MAGIKS!!", \
+		tip_text = "Clear rune", \
+		on_clear_callback = CALLBACK(src, PROC_REF(on_cult_rune_removed)), \
+		effects_we_clear = list(/obj/effect/rune, /obj/effect/heretic_rune, /obj/effect/cosmic_rune), \
+	)
+	AddElement(/datum/element/bane, target_type = /mob/living/simple_animal/revenant, damage_multiplier = 0, added_damage = 25)
 	name = pick(possible_names)
 	desc = possible_names[name]
 
+/obj/item/gun/ballistic/revolver/chaplain/proc/on_cult_rune_removed(obj/effect/target, mob/living/user)
+	SIGNAL_HANDLER
+	if(!istype(target, /obj/effect/rune))
+		return
+
+	var/obj/effect/rune/target_rune = target
+	if(target_rune.log_when_erased)
+		user.log_message("erased [target_rune.cultist_name] rune using a chaplain revolver", LOG_GAME)
+		message_admins("[ADMIN_LOOKUPFLW(user)] erased a [target_rune.cultist_name] rune with a chaplain revolver.")
```

Comment:
this feels too debuggy to be in production code or to be a message admins since it's an intended and natural game mechanic, just delete this and keep the game log only

Reply:
I suppose do you want me to go to the other uses of this proc and remove the admin message?

Reply:
is it like that everywhere? that's a bit stupid but doesn't have to be handled in this PR, but i don't think it's a good idea to replicate here.

Reply:
Already did it sorry. It's kind of in keeping with the pr. I'm here so why not. I'll update the changelog.